### DISCUSSION TITLE
InstallLocation

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-Install#44] [Ideas]
Now, with **InstallLocation** for WAU in registry, we could make sure that the paths are correct (if someone changes them in the WAU installation) in **Winget-Install** when handling the:
- $LogPath
- $WhiteList
- $Mods

**Comment by Romanitho** [comment]
Very true :) haven't think about that haha. I need to change that when I have few minutes. Then, we will have a solid bundle 😋

**Reply by Romanitho** [reply]
#46

**Reply by Romanitho** [reply]
https://github.com/Romanitho/Winget-Install/releases/tag/v1.7.1
